### ISSUE TITLE
uninitialized constant OpscodeAcl::AclBase (NameError)

### ISSUE THREAD

**Issue by scalp42** [chef/knife-acl#30]
```
$> knife group list

/Users/scalp/.rvm/gems/ruby-2.2.4@chef/gems/knife-acl-1.0.1/lib/chef/knife/group_list.rb:26:in `block in <class:GroupList>': uninitialized constant OpscodeAcl::AclBase (NameError)
	from /Users/scalp/.rvm/gems/ruby-2.2.4@chef/gems/chef-12.6.0/lib/chef/knife.rb:225:in `call'
	from /Users/scalp/.rvm/gems/ruby-2.2.4@chef/gems/chef-12.6.0/lib/chef/knife.rb:225:in `block in load_deps'
	from /Users/scalp/.rvm/gems/ruby-2.2.4@chef/gems/chef-12.6.0/lib/chef/knife.rb:224:in `each'
	from /Users/scalp/.rvm/gems/ruby-2.2.4@chef/gems/chef-12.6.0/lib/chef/knife.rb:224:in `load_deps'
	from /Users/scalp/.rvm/gems/ruby-2.2.4@chef/gems/chef-12.6.0/lib/chef/knife.rb:209:in `run'
	from /Users/scalp/.rvm/gems/ruby-2.2.4@chef/gems/chef-12.6.0/lib/chef/application/knife.rb:142:in `run'
	from /Users/scalp/.rvm/gems/ruby-2.2.4@chef/gems/chef-12.6.0/bin/knife:25:in `<top (required)>'
	from /Users/scalp/.rvm/gems/ruby-2.2.4@chef/bin/knife:23:in `load'
	from /Users/scalp/.rvm/gems/ruby-2.2.4@chef/bin/knife:23:in `<main>'
	from /Users/scalp/.rvm/gems/ruby-2.2.4@chef/bin/ruby_executable_hooks:15:in `eval'
	from /Users/scalp/.rvm/gems/ruby-2.2.4@chef/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by stevendanna** [comment]
Looks like a bunch of missing requires. Out of curiosity, are you using knife-rehash?

**Comment by scalp42** [comment]
@stevendanna I am indeed!

**Comment by jeremiahsnapp** [comment]
fixed in 39f507c1f3d2bfc6f544dc5f6d40b7688859793f

knife-acl 1.0.2 is released with the fix. Thanks everyone for helping get this fixed.
